.ci/install_yq.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 SUSE LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+function install_yq() {
+	local yq_path="${GOPATH}/bin/yq"
```

Comment:
I wonder if we should check `GOPATH` is set?
It is a long standing out of date requirement of our CI, as `go` no longer needs `GOPATH` to be set and will default.

Reply:
Yep - you could just source `.ci/lib.sh` which handles this:

- https://github.com/kata-containers/tests/blob/master/.ci/lib.sh#L22..L28

Reply:
I can add a `set -u` to fail if GOPATH is not set. I think for the time being `$GOPATH/bin` is the most obvious destination for the `yq` executable.

Reply:
missed you comment @jodh-intel. I am fine doing that too. It's going to introduce some mutual dependency between the two file, but that shouldn't be circular.